src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -3561,6 +3586,9 @@ public override void PostVisit(DottedNameNode node)
                 _txb.SetStateful(node, _txb.IsStateful(node.Left));
                 _txb.SetContextual(node, _txb.IsContextual(node.Left));
 
+                // An `a.b` expression will be mutable if `a` is mutable
+                _txb.SetMutable(node, _txb.IsMutable(node.Left));
```

Comment:
>_txb.SetMutable(node, _txb.IsMutable(node.Left)); [](http://example.com/codeflow?start=16&length=49)

What happens for expression like,
`First(DataverseTable).OneToManyField`
Should the result be mutable?

Reply:
At this point it is mutable; is there a way to know whether the left node, if it's a FirstNameNode, refers to a local or connected data source?